spec.bs

```diff
@@ -4580,6 +4581,33 @@ Width and height of an ad.
 
 </dl>
 
+<h3 id=direct-from-seller-signals-header>Direct from seller signals</h3>

```

Comment:
Should `direct-from-seller-signals-header` be `direct-from-seller-signals-struct`? This is for the struct, not the header.

Reply:
Thanks, LGTM